### ISSUE TITLE
Faders not working

### ISSUE THREAD

**Issue by Iamskiierman** [ho0ber/nk2-audio#4]
Hey mate, it looks like the actual part of the script that controls the individual faders broke. I'm able to select individual programs (and it writes to the nano.ini file), however I don't have fader / mute control over the channel once it's bound.

I'd love for that to work! Every other part of it is working great (and oddly enough, the master volume fader / mute works just fine).

**Comment by ho0ber** [comment]
Yeah, this is an issue for me too and I've been hearing it from others recently. I haven't had time to work on this recently but rest assured that it's broken even for me.

If anyone else lands on this issue, feel free to take a crack at fixing it and throw up a PR. I can make time to review and test, but at the moment I can't dive in and try to fix it myself.

Also, I am still very slowly working on a complete rewrite (not in autohotkey) that will be easier to maintain.

**Comment by ChrisStein** [comment]
Hey ho0ber, 
I am using nk2 a long time. I also found a solution for the multiple process Id problem for Firefox or Chrome. After I had to do my cable management again my nano kontrol will not be recognized anymore. Now nk2 selects every time my USB audio interface. Is there a possibility to select a midi device? It would be awesome to see a new project without autohotkey. In witch programming language du you want to build it?
Have a great day.

Chris

**Comment by ho0ber** [comment]
Hey @ChrisStein and @Iamskiierman — if you're still having any issues, I've started a new project that is a rewrite of this functionality as a native app instead of a mess of AutoHotkey code. Hopefully, it will be easier for me to address issues as they come up.

There are already some early releases available, and one of the things this new project does is actually enumerate midi devices and pick the nanokontrol. In the near future I plan to allow the user to select devices explicitly as well, as I'll be adding support for other similar hardware.

https://github.com/ho0ber/NK2Tray
